.vib/mysql/goss/goss.yaml

```diff
@@ -0,0 +1,34 @@
+command:
+  mysql-env:
+    exec: env
+    exit-status: 0
+    stdout:
+      - "MYSQL_DATABASE=my_database"
+    stderr: [ ]
+    timeout: 40000  
+  mysql-create:
+    exec: mysql -h localhost -u root -p"$MYSQL_ROOT_PASSWORD" "$MYSQL_DATABASE" -e 'DROP TABLE IF EXISTS TEST; create table TEST( test_id int auto_increment, test_value int, primary key(test_id)); INSERT INTO TEST (TEST_VALUE) VALUES (1993);SELECT * FROM TEST'
```

Comment:
```suggestion
    exec: mysql -h localhost -u "$MYSQL_ROOT_USER" -p"$MYSQL_ROOT_PASSWORD" "$MYSQL_DATABASE" -e 'DROP TABLE IF EXISTS TEST; create table TEST( test_id int auto_increment, test_value int, primary key(test_id)); INSERT INTO TEST (TEST_VALUE) VALUES (1993);SELECT * FROM TEST'
```

Reply:
![Screenshot 2022-06-22 at 11 14 39](https://user-images.githubusercontent.com/47592680/174992074-421f18cc-5ffe-4b0f-989d-70ff21523e36.png)

Seems to me this won't work because the root user is not specified as an environment variable.

Reply:
Oh, then it's okay to leave it as it is 😁